data/functions.sql

```diff
@@ -1071,14 +1071,20 @@ DECLARE
   levels_int INTEGER;
   spaces_per_level INTEGER;
 BEGIN
-  -- if the capacity is set, then use that.
-  IF capacity ~ '^[0-9]+$' THEN
+  -- if the capacity is set to something between 0 and 99999, then use that.
+  IF capacity ~ '^[0-9]{1,5}+$' THEN
     RETURN capacity::integer;
   END IF;
+  -- don't try to do this if way_area is abnormally large.
+  -- (Epcot's parking lot is 647k m^2)
+  IF way_area > 700000 THEN
```

Comment:
just making sure this is the right number.  I understand Epcot's size, but is that the largest parking lot we want to support?

Reply:
Thats about 173 acres which are kinda city block shaped. Generally ETL guards should allow for larger values, just not ridiculously large values. So 2x or 3x the number and call it good?